slurk/views/static/js/connection.js

```diff
@@ -7,7 +7,7 @@ var OV;
 let user_map = {}
 
 function apply_user_permissions(permissions) {
-    $('#type-area').fadeTo(null, permissions.send_message || permissions.send_image || permissions.send_command);
+    $('#type-area').fadeTo(null, permissions.send_message || permissions.private_message || permissions.send_image || permissions.send_command);
```

Comment:
I just noticed that this isn't needed as the user can't send messages with only `permissions.private_message` set.